server/local/pages/Landing.tsx

```diff
@@ -0,0 +1,5 @@
+import { h } from "../../../deps/preact.ts"
+
+export function LandingPage() {
+  return <div>Welcome to the Capi Console</div>
```

Comment:
If there's no styling, I'd rather see a plain text response than HTML.

Reply:
There will be––just a start